src/js/navIcons.js

```diff
@@ -106,10 +130,30 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             pager: ".gpii-fd-navIcon-outer"
         },
         events: {
-            onCreateIcon: null
+            onCreateIcon: null,
+            onSetConfirmedIcon: null,
+            onSetActiveIcon: null
         },
         listeners: {
-            "onCreate.createIcons": "gpii.firstDiscovery.navIcons.createIcons"
+            "onCreate.createIcons": {
+                funcName: "gpii.firstDiscovery.navIcons.createIcons",
+                priority: 3
+            },
+            "onCreate.setInitialIconState": {
+                funcName: "gpii.firstDiscovery.navIcons.setInitialIconState",
+                args: ["{that}"],
+                priority: 2
+            },
+            "onCreate.showIconPage": {
+                funcName: "{that}.showIconPage",
+                priority: 2
```

Comment:
maybe we should use the new priorities for these. Seems that we know what order we want them to come in.